specification/cognitiveservices/data-plane/Speech/VideoDubbing/preview/2024-05-20-preview/video-dubbing.json

```diff
@@ -0,0 +1,849 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Video Dubbing API",
+    "description": "Video dubbing API to translate video.",
+    "contact": {
+      "name": "Azure AI - Speech Services",
+      "url": "https://learn.microsoft.com/azure/ai-services/speech-service/"
+    },
+    "version": "2024-05-20-preview"
+  },
+  "paths": {
+    "/translations": {
+      "get": {
+        "tags": [
+          "Translations"
```

Comment:
"Translation" no "s". Otherwise, in https://editor.swagger.io/ you will find it's not in the same group as get/put/delete.